index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <nav class="navBar">
+        <img src="imgs/My Logo.svg" alt="logo" class="logo">
+        <img src="imgs/Union.svg" alt="menu burger" class="menuIcon">
+        <ul class="navInfo">
+            <li>Portoflio</li>
+            <li>About</li>
+            <li>Contact</li>
+            <li><img src="imgs/message.svg" alt="message incon"></li>
+        </ul>
+    </nav>
+    <main>
+        <div class="mainParagraph">
+            <h1 class="welcomeText">
+                Hey There.
+                I’m Fuhidy.
+            </h1>
+            <h3 class="supportingText">
+                I'm a Software Developer
+            </h3>
+            <p class="description">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you
+                see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+        </div>
+        <div class="social-media">
+            <a href="#"> <img src="imgs/social-media/github.svg" alt="github logo"></a>
+            <a href="#"> <img src="imgs/social-media/linkedin.svg" alt="linkedin logo"></a>
+            <a href="#"> <img src="imgs/social-media/angiel.svg" alt="angelist logo"></a>
+            <a href="#"> <img src="imgs/social-media/twitter.svg" alt="twitter logo"></a>
+            <a href="#"> <img src="imgs/social-media/medium.svg" alt="medium logo"></a>
+        </div>
```

Comment:
- [ ] This is the list of the icons and for that, we use the `ul` tag to group them `li` per icon and each icon has a link (`a` tag) has is done. please implement it